src/main/java/com/gd/clinic/entities/Patient.java

```diff
@@ -0,0 +1,53 @@
+package com.gd.clinic.entities;
+
+import lombok.*;
+import org.hibernate.Hibernate;
+import org.hibernate.annotations.Type;
+
+import javax.persistence.*;
+import java.util.Objects;
+import java.util.UUID;
+
+
+@RequiredArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@ToString
+@Entity
+@Table(name = "Patients")
+public class Patient{
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Type(type = "uuid-char")
+    private UUID id;
+
+    @NonNull
+    private String name;
+
+    @NonNull
+    private String diagnosis;
+
+    @NonNull
+    private String insuranceNumber;
+
+    @Type(type = "uuid-char")
+    private UUID doctorId;
+
+    private String status;
+
```

Comment:
it should also contain `createdAt` and `createdBy`

Reply:
Makes sense, added those fields to the class. Since security isn't implemented in this branch, some code is commented out. This is specifically for the `createdBy` field. I also created a method to get the current user for this field, not sure if this is the best way to get that value from the `SecurityContextHolder`, I mean just a `String` containing the username...